d2ir/d2ir.go

```diff
@@ -601,6 +601,18 @@ func (m *Map) EdgeCountRecursive() int {
 	return acc
 }
 
+func (m *Map) GetClosestClassMap(name string) *Map {
```

Comment:
```suggestion
func (m *Map) GetClassMap(name string) *Map {
```

This doesn't recurse up so Closest isn't accurate.